src/site/content/en/blog/gpu/index.md

```diff
@@ -0,0 +1,233 @@
+---
+title: Access modern GPU features with WebGPU
+subhead: |
+  WebGPU enables high-performance 3D graphics and data-parallel computation on
+  the Web.
+authors:
+  - beaufortfrancois
+  - cwallez
+date: 2021-08-26
+#updated: 2021-08-26
+hero: image/vvhSqZboQoZZN9wBvoXq72wzGAf1/SN6GIsxmcINXJZKszOTr.jpeg
+thumbnail: image/vvhSqZboQoZZN9wBvoXq72wzGAf1/SN6GIsxmcINXJZKszOTr.jpeg
+description: |
+  WebGPU enables high-performance 3D graphics and data-parallel computation on
+  the Web.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - capabilities
+  - games
+feedback:
+  - api
+stack_overflow_tag: webgpu
+---
+
+## What is WebGPU? {: #what }
+
+[WebGPU] is a new API for the web that exposes modern computer graphics APIs
+such as Direct3D 12, Metal, and Vulkan for performing rendering and computation
```

Comment:
Is this list comprehensive or just examples? 'Such as' made me think examples. The diagram makes me think comprehensive. Please clarify this.

Reply:
This list is comprehensive.

Reply:
I have a new suggestion that clarifies that.

Reply:
What is the proper way to mention those APIs then?